### ISSUE TITLE
Desktop(Windows, macOS, Linux) Support

### ISSUE THREAD

**Issue by friebetill** [mixpanel/mixpanel-flutter#36]
Are there any plans to support the Windows version?

**Comment by zihejia** [comment]
hi @friebetill , we currently don't have a plan to support Desktop apps unless there are more requests from the community. Thanks for bringing it up.

**Comment by kupelimurat** [comment]
hi, we need windows support

**Comment by edwardaux** [comment]
Just one more vote for Desktop support... I've got an app that is targeting iOS, Android and desktop, but unless I've got an common analytics provider that works for all of them, then it likely means choosing another provider.

In the meantime, do you have a recommended workaround? Anyone know if https://github.com/koa-health/mixpanel_analytics works and is reliable?

**Comment by pawan2601** [comment]
We need Windows and Mac support for Mixpanel Flutter.

**Comment by marconemagnus** [comment]
Windows support would be really good, I gave up using analytics for Firebase because it doesn't have Windows support. I wish I didn't have to do the same with Mixpanel. 🥺

**Comment by marconemagnus** [comment]
> Just one more vote for Desktop support... I've got an app that is targeting iOS, Android and desktop, but unless I've got an common analytics provider that works for all of them, then it likely means choosing another provider.
> 
> In the meantime, do you have a recommended workaround? Anyone know if https://github.com/koa-health/mixpanel_analytics works and is reliable?

@edwardaux have you tried  [https://github.com/koa-health/mixpanel_analytics](https://github.com/koa-health/mixpanel_analytics ) ?  would you recommend using it?

**Comment by edwardaux** [comment]
@marconemagnus Yep, I ended up using that package. My analytics requirements are pretty modest at the moment, but it has been great so far.

**Comment by markbreuss** [comment]
+1 for Desktop (Windos & MacOs) Support

**Comment by zihejia** [comment]
Thanks for all the feedback. We will consider adding Desktop Support soon and will give you the update in terms of the timeliine.

**Comment by marconemagnus** [comment]
Hi @zihejia , any chance you can give us a timeline on this? 

I'm using a third party lib that works like a wrapper around Mixpanel's API, but it doesn't give me the default capabilities that your SDK does. It does work as a temporary workaround though.

thank you I appreciate you attention.

**Comment by jonorozcoc** [comment]
Hi @zihejia ,

I would like to know if there is any update on this? Many of us want to use Mixpanel on desktop apps

**Comment by ihadabs** [comment]
Hi @zihejia,

Any update on the desktop support timeline?

**Comment by vlazdra** [comment]
> We will consider adding Desktop Support soon and will give you the update in terms of the timeliine.

Hey @zihejia, any updates on the timeline? I think the community would very much appreciate this feature.

**Comment by satvikpendem** [comment]
I'm also interested in desktop support, especially as Flutter was made for supporting all of the major application paradigms, mobile, web and desktop, so this would be useful.

**Comment by Jethro87** [comment]
macOS support for Flutter is really needed. I'm about to release a mac app and am now not sure what I'll do re: analytics going forward.

**Comment by robhageboeck** [comment]
Windows support desired here. The 3rd party wrapper is fine but doesn't cover automatic events. Having to roll the current automatic events into manually tracked events would be a bunch of work and would remove the usefulness of the flutter mobile sdk for organizations forced to do it. Maybe a the desktop projects could be created and stubbed out to show some progress towards desktop support in flutter?

**Comment by Wizzel1** [comment]
Yes, please add desktop support. This would give you a strong competitive advantage

**Comment by yelkamel** [comment]
Any update for desktop support ?

**Comment by sampontec** [comment]
Support for desktop would be much appreciated. Any updates on timeline?

**Comment by AndreyKuzmenko** [comment]
+1

**Comment by sundong212** [comment]
Vote +1 for Windows support. Appreciated!

**Comment by MousyBusiness** [comment]
There is very minimal work required to get a MacOS version out. I'd say that is the bare minimum when you already have an iOS app.

**Comment by marconemagnus** [comment]
I gave up, no longer using Mixpanel because of this.

**Comment by Wizzel1** [comment]
Same here, you should check out https://www.june.so/
There is a flutter package for that: https://pub.dev/packages/june_flutter

**Comment by rienkvr** [comment]
Was switching to MixPanel assuming this was taken care of. It really is a requirement for an analytics tool to support all the platforms.

**Comment by prinkeeras** [comment]
Would love to see support for Mac and windows

**Comment by dickverweij** [comment]
Me too Me too

**Comment by lukemmtt** [comment]
+1 for macOS support

**Comment by walid-ashik** [comment]
still there's no macOS support? at least you guys can give us a workaround if we want to support it. 
I really need macOS support!

**Comment by lukemmtt** [comment]
+1 for macOS support

**Comment by ttzein6** [comment]
+1 for macOS and Windows support

**Comment by jpv123** [comment]
Any updates on this? @friebetill

**Comment by jpv123** [comment]
Any updates on this? @zihejia

**Comment by zhujiageng** [comment]
+1 for Windows support

**Comment by jingyuan2017** [comment]
+1 Windows / macOS 
Thanks.

**Comment by Bahrom2101** [comment]
How many more requests are needed?

**Comment by suxoikorm** [comment]
+1 for Windows and macOS support

**Comment by melbek** [comment]
+2 for Desktop support
